db/migrate/20231123035111_create_books.rb

```diff
@@ -3,9 +3,9 @@
 class CreateBooks < ActiveRecord::Migration[7.1]
   def change
     create_table :books do |t|
-      t.string :name, presence: true
+      t.string :name, null: false
```

Comment:
For this one, if this project already has staging or production. This way will not work because rails will detect that this migration has already migrated. So, changes in this file will not make any update to database.
Solution is to create new migration file and add logic to new migration instead